tests/components/irm_kmi/test_coordinator.py

```diff
@@ -0,0 +1,23 @@
+"""Tests for the coordinator of IRM KMI integration."""
+
+from datetime import timedelta
+from unittest.mock import MagicMock
+
+from homeassistant.components.irm_kmi.coordinator import IrmKmiCoordinator
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+def test_update_interval_is_7_minutes(
+    hass: HomeAssistant,
+    mock_config_entry: MockConfigEntry,
+) -> None:
+    """Test that the refresh interval 7 minutes.
+
+    If you need to change this test, also change the documentation: https://github.com/home-assistant/home-assistant.io.
+    """
+
+    coordinator = IrmKmiCoordinator(hass, mock_config_entry, MagicMock())
```

Comment:
Meh, generally we don't have tests for this. If you want to implicitly test this, add a test to the weather platform where you update, freeze time and skip 7 minutes and observe it updated again

Reply:
Got it!  I used this test as a safeguard when developing to avoid pushing code with a low refresh interval.  I guess it is not needed anymore.  Removed it in 54c7745d515